File: solutions/cpp/Is Subsequence.cpp
#include <iostream>
class Solution
{
public:
    bool isSubsequence(std::string s, std::string t)
    {
        int i = 0, j = 0, count = 0;
        while (i < s.length() && j < t.length())
        {
            if (s[i] == t[j])
            {
                count++;
                i++;
            }
            j++;
        }
        if (count == s.length())
            return true;
        return false;
    }
};

File: solutions/python/Is Subsequence.py
class Solution:
    def isSubsequence(self, s: str, t: str) -> bool:
        [i,j,count]=[0,0,0]
        while i<len(s) and j<len(t):
            if s[i]==t[j]:
                count=count+1
                i=i+1
            j=j+1
        if count==len(s): return True
        return False